auth-web/src/components/auth/account-settings/team-management/InviteUsersForm.vue

```diff
@@ -149,6 +148,14 @@ export default class InviteUsersForm extends Vue {
     return new Set(invitations.map(invitation => invitation.emailAddress.toLowerCase())).size !== invitations.length
   }
 
+  private get isAccountGovM () : boolean {
```

Comment:
can this go to a common place please? like a mixin or something which can be reused